test/spec/coreSpec.js

```diff
@@ -1,730 +1,1801 @@
 var obj;
 if (typeof jsonpatch === 'undefined') {
-    if (process.env.DUPLEX === 'yes') { //required by `jasmine-node` test runner in Node.js
-        jsonpatch = require('./../../src/json-patch-duplex.js');
-    } else {
-        jsonpatch = require('./../../src/json-patch.js');
-    }
+  if (process.env.DUPLEX === 'yes') {
+    //required by `jasmine-node` test runner in Node.js
+    jsonpatch = require('./../../src/json-patch-duplex.js');
+  } else {
+    jsonpatch = require('./../../src/json-patch.js');
+  }
 }
 if (typeof _ === 'undefined') {
-    _ = require('./../lib/underscore.min.js');
+  _ = require('./../lib/underscore.min.js');
 }
+describe('using with patch.reduce', function() {
```

Comment:
I'd describe it as `jsonpatch.applyReducer - using  with Array#reduce`  as just `patch` is not so explicit